web/yo/app/scripts/services/databaseconnector.js

```diff
@@ -43,6 +45,28 @@ angular.module('oncokbApp')
             var testing = OncoKB.config.testing || false;
             var inProduction = OncoKB.config.production || false;
 
+
+            function getAllDrugs() {
+                var deferred = $q.defer();
+                Drugs.getAllDrugs()
+                    .then(function(data) {
+                        deferred.resolve(data.data);
+                    }, function(error) {//send email?
+
```

Comment:
If the endpoint failed, we should display an error message at the place you want to show the drugs table.